vertical-pod-autoscaler/pkg/admission-controller/logic/recommendation_provider.go

```diff
@@ -60,6 +61,19 @@ func NewRecommendationProvider(vpaLister vpa_lister.VerticalPodAutoscalerLister,
 	}
 }
 
+func limitMatchesRequest(limit, request *resource.Quantity) bool {
+	// Limit not set, it cannot match the request.
+	if limit == nil || limit.Value() == 0 {
+		return false
+	}
+	// Limit set but request not set - K8s interprets will treat the pod as if it had them equal.
```

Comment:
nit: grammar seems broken in "K8s interprets..."

Reply:
Fixed.